UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -890,7 +890,11 @@ protected virtual void BuildAnimationSamplers(GLTFAnimation animation, int anima
 				var samplerDef = animation.Samplers[i];
 
 				// set up input accessors
-				BufferCacheData bufferCacheData = _assetCache.BufferCache[samplerDef.Input.Value.BufferView.Value.Buffer.Id];
+
+				var inputBuffer = samplerDef.Input.Value.BufferView.Value.Buffer;
```

Comment:
Doesn't look like this new variable is used

Reply:
Good catch.  I've removed it now and changed it to use two separate variables for input and output bufferCacheData rather than reusing the same variable.